chart/templates/scheduler/scheduler-serviceaccount.yaml

```diff
@@ -23,7 +23,6 @@
 {{- if and .Values.scheduler.enabled .Values.scheduler.serviceAccount.create }}
 apiVersion: v1
 kind: ServiceAccount
-automountServiceAccountToken: {{ .Values.scheduler.serviceAccount.automountServiceAccountToken }}
```

Comment:
These are the possible cases:
1. We can set it to true for cases when it is not CeleryExecutor
2. For other cases it has to be true or the key can be removed entirely.

Two possible approaches. Either control addition or removal of key based on the executor, otherwise allow to override only for CeleryExecutor